typescript/sdk/src/aws/validator.ts

```diff
@@ -32,36 +36,36 @@ export class S3Validator extends BaseValidator {
   static async fromStorageLocation(
```

Comment:
nit: I'm don't feel strongly about this but it may be cleaner to define two separate methods, one that expects the s3 url, the other that expects https

Reply:
the way I thought about it is that since it's the same parameter used for both path types, and there's also the check in `announce-validators.ts` that hints at the function being used for both path types, this is fine. I also don't feel strongly but since it's still backwards compatible I guess I'll wait for the next person to complain if this needs changing :P